app/routes/wow.item-data.$itemId.tsx

```
@@ -15,6 +16,42 @@
 import SmallTable from '~/components/WoWResults/FullScan/SmallTable'
 import CustomButton from '~/components/utilities/CustomButton'
 import Banner from '~/components/Common/Banner'
+import { useState, useEffect, useMemo, memo, Suspense } from 'react'
+import { Transition } from '@headlessui/react'
+
+// Loading skeleton component
+const LoadingSkeleton = () => (
+  <PageWrapper>
+    <div className="animate-pulse">
+      <div className="h-8 bg-gray-300 dark:bg-gray-600 rounded mb-4 w-1/3"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)